react_practice/react-apps/waitlist-page-v1/src/index.css

```diff
@@ -1,3 +1,7 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
I added this global style to your app that corrects some of the "math" on how the size of elements are determined. The default box-sizing treats sizing a component in a very un-intuitive way.

I'd highly encourage you to take a look at [this article](https://css-tricks.com/box-sizing/) which does a much better job than I can do in a comment box on how this affects the sizing of your elements.